lib/modules/versioning/pvp/index.ts

```diff
@@ -112,7 +112,7 @@ function getNewValue({
   newVersion,
   rangeStrategy,
 }: NewValueConfig): string | null {
-  if (rangeStrategy !== 'auto') {
+  if (rangeStrategy !== 'widen') {
```

Comment:
But shouldn't `auto` also be ok?

Reply:
It seems that `auto` should never be sent to `getNewValue()`. Only two other versioning modules have `auto` awareness and both of them just default it to `replace`. On the other hand, the existence of this `if` statement until now implies that `auto` was maybe being passed?

Reply:
> the existence of this if statement until now implies that auto was maybe being passed?

The previous behavior in PVP versioning was tested in conjunction with a draft Cabal manager that would return `auto` in `getRangeStrategy`. In this configuration, `auto` *would* get to versioning. But it doesn't make sense to do have the manager return `auto`, so when I made this PR and double-checked, tested and reviewed the code, I noticed the issue. That's why it's only getting fixed now, even though it is in versioning, and not the manager.